go.mod

```diff
@@ -26,6 +26,19 @@ require (
 	gopkg.in/yaml.v3 v3.0.1
 )
 
+require (
+	github.com/gkampitakis/ciinfo v0.3.0 // indirect
```

Comment:
Are these separate in the go.mod file because it's just for the test dependency? (If so can you add a comment above the require)

Reply:
I have not done anything manually with `go.mod`, so this was all generated by `go mod tidy` - I'll see if just moving these into the other `require` makes it happy, otherwise I'd prefer not to touch (or comment) it since it's using some internal go semantics

Reply:
ok yup it just seems to be `go.mod` preserves groups if you create them, and I must have somehow (probably during a rebase or something)